faq/index.html

```diff
@@ -124,17 +124,18 @@
                         <div class="faq-content-wrapper markdown">
                             <p>Here are our plans for the future versions of Spine:</p>
                             <ul>
-                                <li><b>Release version 1.0;</b></li>
-                                <li><b>Implement Catch-Up for Projections in Java.</b></li>
+                                <li>Migrate to <b>Java 11;</b></li>
+                                <li><b>Scaffold</b> entity classes from Protobuf;</li>
+                                <li><b>Generate Bounded Contexts</b> using existing Java model definitions;</li>
+                                <li><b>Release version 2.0.</b></li>
```

Comment:
We spoke about v1.0 earlier to tell people that making it stable for being used in real projects is a priority for us.
v2.0 is a different thing. We may group features starting from Java 11 as v2, but it's not of much importance in the context of FAQs. We probably need a bit more detailed picture of priorities similarly to [how our colleagues from JxBrowser team do this](https://jxbrowser-support.teamdev.com/roadmap/). But that's a topic of a bigger discussion. For now, please remove v2 item.